data/remnant missions.txt

```diff
@@ -941,7 +941,59 @@ mission "Remnant: S-970 Regenerator"
 
 
 
-mission "Remnant: taely intro log"
+mission "Remnant: Wanderer Ramscoop"
+	name "Retrieve Wanderer Ramscoops for the Remnant"
+	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two ramscoops from the Wanderers for them to study."
+	minor
+	source
+		government "Remnant"
+	destination "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		require "Wanderer Ramscoop"
+		conversation
+			`Strolling into the starport, you wonder if someone from the nosy gull would be approaching you. While you have made the approach to <planet> many times, this time was differentiated by a gull that seemed to be shadowing you closely. Just looking at it, you guessed that it was probably a reconnaissance ship, or possibly a science vessel. Few other ships would carry quite so much scanning equipment.`
+			`	Looking around, you quickly notice a young Remnant trotting into the area from a different landing pad. He spots you and approaches, then clears his throat nervously "Greetings, traveler" he warbles, as if unused to singing. "We scanned you as you flew by, and noticed some unusual technology mounted to your hull. Can you tell us what they are?" With this, he pulls up his comlink and shows you an image of what appear to be your ramscoops.`
+			choice
+				`	"Those are wanderer ramscoops."`
+				`	"I'd rather not share that."`
+					decline
+			
+			`	"They are?" He looks disappointed. "I guess I'll be taking Mala's next watch duty, then. She guessed that they were ramscoops, I thought they were a scanning array." He looks slightly disappointed, but quickly regains his usual impassivity. "You said 'Wanderer' ramscoops? Who are these 'Wanderers?' Are they a group like us, hidden away somewhere?"`
+			choice
+				`	"Wanderers are an alien race that live in counter-spinward from humanity."`
+				`	"I would prefer to not talk about them."`
+					goto notalk
+			`	He listens to your explanation calmly, but his tone takes on a note of eagerness as he chants "These Wanderers sound very interesting. Thank you for telling me about them. It seems that we might have much that we could learn from them." He looks thoughtful for a moment before continuing.`
+			label notalk
+			`	"Be that as it may, the opportunity to study the fuel generation systems from someone with a different technological path would be most advantageous. If you could procure two of these 'Wanderer Ramscoops' for us, we would reimburse you for the cost, plus 750000 credits for your efforts."`
+			choice
```

Comment:
This choice only makes sense after the `notalk` label text, but doesn't work as well when it's immediately after the other option, since in that case the guy hasn't actually asked you to do anything specific to which you could respond `Sure, I could do that`.